synapse/config/workers.py

```diff
@@ -50,13 +52,42 @@ def _instance_to_list_converter(obj: Union[str, List[str]]) -> List[str]:
     return obj
 
 
-@attr.s(auto_attribs=True)
-class InstanceLocationConfig:
+class InstanceLocationConfigModel(BaseModel):
```

Comment:
I think this ought to just be `ConfigModel`, somewhere general in `synapse.config`. (If it was bespoke to this model then we could just define `Config` directly as a subclass of `InstanceLocationConfig`.

Reply:
Perhaps that new file to hold new Pydantic `Model`s should be sooner rather than later. Unless you think `config/_util.py` is appropriate for now?

(+1 to a rename to `ConfigModel`)

Reply:
Renamed in [28f03ce](https://github.com/matrix-org/synapse/pull/15431/commits/28f03ce806cfd0662be754af3be94f4c78238060)